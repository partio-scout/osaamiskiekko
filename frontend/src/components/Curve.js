import React from 'react';
import styled from 'styled-components';
import Curve from '../images/curve.svg';
import SearchBox from './SearchBox';

const S = {};
S.Curve = styled.div`
.curve-container {
  margin: auto;
  position: relative;
}

.curve-image {
  width: 100%;
}

  @media only screen and (max-width: 600px) { 
    .search-container {
      top: 65px;
      max-width: 250px;
    }
  }
`;

export default function Header() {
  return (
    <S.Curve>
      <div className="curve-container">
        <SearchBox />
        <img src={Curve} alt="Curve" className="curve-image" />
      </div>
    </S.Curve>
  )
}
