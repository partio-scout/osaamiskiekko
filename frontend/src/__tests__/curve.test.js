import React from 'react';
import renderWithProviders from '../testHelpers'
import Curve from '../components/Curve';

test('Curve snapshot matches', () => {
  const tree = renderWithProviders(
        <Curve />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
