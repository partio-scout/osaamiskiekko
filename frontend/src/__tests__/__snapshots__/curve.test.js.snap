// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Curve snapshot matches 1`] = `
<div
  className="sc-ifAKCX fpVbKk"
>
  <div
    className="curve-container"
  >
    <div
      className="sc-htpNat dFBUdP"
    >
      <div
        className="sc-bxivhb hywrlk"
      >
        <label
          htmlFor="search-academy"
        >
          <span>
            Valitse järjestö tai oppilaitos
          </span>
        </label>
        <span
          className="fa fa-search"
        />
        <input
          name="search-academy"
          onBlur={[Function]}
          onChange={[Function]}
          onKeyDown={[Function]}
          placeholder="Hae..."
          type="text"
          value=""
        />
      </div>
      <div
        className="sc-bwzfXH jsMlNV"
      />
    </div>
    <img
      alt="Curve"
      className="curve-image"
      src="curve.svg"
    />
  </div>
</div>
`;
